src/basic/selinux-util.c

```diff
@@ -244,9 +244,14 @@ int mac_selinux_apply(const char *path, const char *label) {
         assert(label);
 
         if (setfilecon(path, label) < 0) {
-                log_enforcing_errno(errno, "Failed to set SELinux security context %s on path %s: %m", label, path);
+                const int saved_errno = errno;
+
+                /* Check for policy change so enforcing state is kept up-to-date by callbacks */
+                (void) avc_netlink_check_nb();
```

Comment:
wouldn't it make sense to fold this into the mac_selinux_enforcing() call?

Reply:
yes, refactored